qiskit_nature/circuit/library/ansatzes/ucc.py

```diff
@@ -33,10 +33,11 @@
 class UCC(EvolvedOperatorAnsatz):
     r"""The Unitary Coupled-Cluster Ansatz. For more information, see [1].
 
-    This Ansatz is an :class:`~.EvolvedOperatorAnsatz` given by :math:`e^{T - T^{\dagger}}` where
+    This Ansatz is an `EvolvedOperatorAnsatz` given by :math:`e^{T - T^{\dagger}}` where
     :math:`T` is the *cluster operator*. This cluster operator generally consists of excitations
```

Comment:
```suggestion
    :math:`T` is the *cluster operator*. This cluster operator generally consists of excitation
```

Reply:
@mrossinek This cluster operator generally consists of a sum of excitations. I do not understand your suggestion.

Reply:
It says: `excitations operators` which does not work. I suggest it to be `excitation operators`

Reply:
ok now I see. thanks